src/api/app/views/webui/user/login.html.haml

```diff
@@ -0,0 +1,11 @@
+- @pagetitle = "Please Log In"
+- if CONFIG['proxy_auth_mode'] == :on
+  %form{ action: "#{CONFIG['proxy_auth_login_page']}", enctype: "application/x-www-form-urlencoded", method: "post" }
+    %div
+      %input{name: "context", type: "hidden", value: "default"}/
+      %input{name: "proxypath", type: "hidden", value: "reverse"}/
+      %input{name: "message", type: "hidden", value: "Please log In"}/
+      = render partial: 'login_form'
+- else
+  = form_tag controller: :user, action: :do_login, method: :post do
```

Comment:
Because before they just put the tags for opening and closing the form independently... now in HAML if you want it to close the tag you need to pass the block, I moved the  inner code to a partial, so the `do` is needed